queue/queue.py

```diff
@@ -1,18 +1,94 @@
-from doubly_linked_list import ListNode
-from doubly_linked_list import DoublyLinkedList
+"""Each ListNode holds a reference to its previous node
+as well as its next node in the List."""
+class ListNode:
+    def __init__(self, value, prev=None, next=None):
+        self.value = value
+        self.prev = prev
```

Comment:
good thinking! this is only needed for Doubly Linked Lists, but it's great that you're thinking about this.